### ISSUE TITLE
Email Templates

### ISSUE THREAD

**Issue by jackcarlisle** [karmaradio/karma#43]
At the moment our email templates are very basic. Once we have the branding/styling for the application we'll be able to design the email templates so that they are consistent with said styling.

**Comment by JMurphyWeb** [comment]
@jonscottfilm could we get the email copy please?

Emails:
Verify email address, 
Invited to join application (offer made),
updated project offer,
Reset password

**Comment by JMurphyWeb** [comment]
@jonscottfilm assigning you, copy is main priority for this.

**Comment by jonscottfilm** [comment]
On it.

**Comment by jonscottfilm** [comment]
Where and in what format do you want these once I've written them?

**Comment by jonscottfilm** [comment]
OK, this is 4 out of 6 email templates, copy & formatting. (Contractor responses to do tomorrow)

This will serve purpose at the point we have HTML-formatted email notifications, but if it's just plain text emails for now, please note capitalisation and bold for emphasis where applicable.

This is in Word, so you can copy as is, or just take as plain text, however you need.

Merged fields are in curly brackets, e.g {Project Codename}. 

Would prefer to embed links if possible. 
(do we also need to paste the full links in, perhaps beneath the email signature, i.e. 'if link not working, please copy and paste: LINK IN FULL) ?

[Engine notification emails text copy format.docx](https://github.com/karmaradio/karma/files/1023822/Engine.notification.emails.text.copy.format.docx)

**Comment by JMurphyWeb** [comment]
4 out of 6 templates are in @jonscottfilm

**Comment by jonscottfilm** [comment]
Inconsistent use of full stops (I know!), across templates. Could have been me...

Guess we have them.

**Comment by jonscottfilm** [comment]
I need to rewrite some of these templates. Too wordy :)
And need slightly better styling.

**Comment by jackcarlisle** [comment]
Ok no problem, let us know about the styling of them. Is this something that you would like @nogainbar 's opinion for?

**Comment by jonscottfilm** [comment]
When I write the 2 remaining email templates (hopefully soon!) I'll have a go at re-styling too. And if not, add it to @nogainbar's pile... :)

**Comment by jonscottfilm** [comment]
:) Little premature. I need to finish this for Sp2, too.

**Comment by JMurphyWeb** [comment]
@jonscottfilm would be great to get this sent through in the next day so we can close this sprint off.

**Comment by jonscottfilm** [comment]
All on @jonscottfilm.
 
I will have ALL of these, written and all reformatted in pretty HTML, by Wed 28 Jun. :)
